internal/slices/slices.go

```diff
@@ -25,3 +25,13 @@ func Contains[E isser](s []E, name string) bool {
 func Delete[S ~[]E, E isser](s S, i int) S {
 	return append(s[:i], s[i+1:]...)
 }
+
+// Contains reports whether v is present in s.
+func ContainsGenerics[E comparable](s []E, v E) bool {
```

Comment:
nit: The function name is acceptable but still weird. It should named as `Contains` but the name is occupied. How about `ContainerElement`?